stashpr.py

```diff
@@ -43,14 +43,16 @@ def find_username():
 
 
 def find_description(frm, to):
-    return to
+    to = to if '/' in to else 'origin/%s' % to
+    return execute(['git', 'show', '-s', '--format=%B', '%s...%s' % (frm, to)])
```

Comment:
the `...` revrange always confuses me.. maybe it would be better write as `branch ^target` which should be the thing we want. e.g all commits in branch not in the target of the pull-request